lib/typescript/render/providers/facebook/FacebookRender.tsx

```diff
@@ -109,8 +133,8 @@ const parseAttachment = (attachment: SimpleAttachment | ButtonAttachment | Gener
   if (attachment.type === 'fallback') {
     return {
       type: 'fallback',
-      title: attachment.payload.title,
-      url: attachment.payload.url,
+      title: attachment.payload?.title ?? attachment.title,
+      url: attachment.payload?.url ?? attachment.url,
     };
   }
```

Comment:
The fallback attachment does not always include the` title` and `url` in the `payload`: 
- https://developers.facebook.com/docs/messenger-platform/reference/webhook-events/message-echoes/#fallback
- https://developers.facebook.com/docs/messenger-platform/reference/webhook-events/messages/